src/main/webapp/view/transcript/transcript.js

```diff
@@ -172,3 +172,15 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Resets the transcript container to what it looks like before it
```

Comment:
This comment is a little confusing can you please rephrase this

Reply:
Also PR name